cs-office-365/src/main/java/io/cloudslang/content/office365/services/UserServiceImpl.java

```diff
@@ -53,6 +55,38 @@
         return new HttpClientService().execute(httpClientInputs);
     }
 
+    @NotNull
+    public static Map<String, String> getUser(@NotNull final GetUserInputs getUserInputs) throws Exception {
+        final HttpClientInputs httpClientInputs = new HttpClientInputs();
+        final Office365CommonInputs commonInputs = getUserInputs.getCommonInputs();
+        httpClientInputs.setUrl(getUserUrl(getUserInputs));
+
+        HttpCommons.setCommonHttpInputs(httpClientInputs, commonInputs);
+
+        httpClientInputs.setAuthType(ANONYMOUS);
+        httpClientInputs.setMethod(GET);
+        httpClientInputs.setKeystore(DEFAULT_JAVA_KEYSTORE);
+        httpClientInputs.setKeystorePassword(CHANGEIT);
+        httpClientInputs.setContentType(APPLICATION_JSON);
+
+        httpClientInputs.setResponseCharacterSet(commonInputs.getResponseCharacterSet());
+        httpClientInputs.setHeaders(getAuthHeaders(commonInputs.getAuthToken()));
+
+        return new HttpClientService().execute(httpClientInputs);
+    }
+
+    @NotNull
+    private static String getUserUrl(@NotNull final GetUserInputs getUserInputs) throws Exception {
```

Comment:
Maybe this method should have as arguments String userPrincipalName and String userId

Reply:
addressed the changes